isolate_example/README.md

```diff
@@ -9,6 +9,24 @@ A sample application that demonstrate best practices when using [`isolates`](htt
 * Demonstrate how to initialize and use an isolate.
 * Show the cost of moving data between isolates and provide alternatives.
 
+## The important bits
+
+### `page_one.dart`
+
+Compares running a large computation on the main isolate with running the same calculation on a second isolate.
+When the main isolate is used, Flutter is unable to render new frames, and the `SmoothAnimationWidget`'s animation freezes.
+### `page_two.dart`
+
+Creates an isolate used to compute an infinate loop of sum of 100M randomly generated numbers while reporting progress. 
```

Comment:
Is "infinite loop" accurate here? It's a long-running loop, certainly, but it does eventually exit.

Consider "...used to compute the sum of 100M..." instead.

Reply:
I believe that infinite loop is accurate since it will run forever unless the user presses terminate

Reply:
Discussed offline, and "infinite loop" is indeed more accurate.